server/test/article.spec.js

```diff
@@ -169,5 +169,13 @@ describe('Articles routes', () => {
       expect(res).to.have.status(422);
       expect(message).to.be.equal('title, body, description, tags and published fields are required');
     });
+    it('it should get all articles', async () => {
```

Comment:
Not all test coverage has been met.

Reply:
> Not all test coverage has been met.

I'm not testing for when no article is in the database because there will always be articles in the database. Moreover, I tried it and the test was failing because the database was not empty.

Reply:
Ok.